app/components/repo-actions.js

```diff
@@ -1,7 +1,7 @@
 import Component from '@ember/component';
 import { inject as service } from '@ember/service';
 import { computed } from '@ember/object';
-import { alias, and, not, or } from '@ember/object/computed';
+import { alias, and, not, or, equal } from '@ember/object/computed';
```

Comment:
This change seems unnecessary

Reply:
Done, Rest @artursmirnov a small fix is also provided in this PR, which is to take care of priority status after the success of prioritizing build.